src/pages/error.jsx

```
@@ -1,22 +1,30 @@
 import { Header } from '../components/utils/header';
 import { Footer } from '../components/utils/footer';
-
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
 
 export default function Error() {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)